getting-started/README.adoc

```diff
@@ -212,6 +212,8 @@ endif::[]
 ifdef::qs[]
 [#conclusion]
 Congratulations! You successfully completed the {product} Getting Started quick start, and are now ready to use the service.
+
+You can use Kafka scripts to check that you can connect with your Kafka instance.
```

Comment:
@jschuler Wondering if we can have 2 x _next quick start_ links from this quick start. We could then suggest Kafkacat here too.